### PR TITLE
feat: List 2 no pic 6 tablet

### PR DESCRIPTION
**Pull request by aronshamash**
- as per https://nidigitalsolutions.jira.com/browse/REPLAT-5245

### Android tablet Nexus landscape
screenshot 2019-02-27 at 16 27 10

### CODE REVIEW

File: packages/edition-slices/src/tiles/tile-l/styles/index.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ const styles = {
   },
   headlineStyle: {
     fontFamily: fonts.headline,
-    fontSize: 22,
-    lineHeight: 22,
+    fontSize: 20,
```
Comment (Ivaylo1987) [inline packages/edition-slices/src/tiles/tile-l/styles/index.js]: Isn't this going to change for the mobile breakpoint, too?

Comment (aronshamash) [reply packages/edition-slices/src/tiles/tile-l/styles/index.js]: yes it was incorrect before. both mobile and tablet should be 20 according to zeplin

Comment (Ivaylo1987) [reply packages/edition-slices/src/tiles/tile-l/styles/index.js]: LGTM 🐷

File: packages/edition-slices/src/tiles/tile-l/styles/index.js [hunk 2]
```diff
@@ -7,8 +7,8 @@ const styles = {
   },
   headlineStyle: {
     fontFamily: fonts.headline,
```
Comment (jimhunty) [inline packages/edition-slices/src/tiles/tile-l/styles/index.js]: Should this use the `fontFactory`?

File: packages/edition-slices/src/slices/listtwoandsixnopic/index.js [hunk 3]
```diff
@@ -15,34 +16,41 @@ const ListTwoAndSixNoPicSlice = ({
...
+        <TileL onPress={onPress} tile={support5} tileName="support5" />
+      )}
+      renderSupport6={() => (
+        <TileL onPress={onPress} tile={support6} tileName="support6" />
+      )}
+    />
+  );
+
+  return (
+    <ResponsiveSlice renderMedium={renderSlice} renderSmall={renderSlice} />
```
Comment (jimhunty) [inline packages/edition-slices/src/slices/listtwoandsixnopic/index.js]: How does breakpoint get read in this case? In storybook should the render function have `renderMedium(breakpoint)`?

File: packages/edition-slices/src/tiles/tile-l/styles/index.js (hunk 2 above)
Comment (aronshamash) [reply packages/edition-slices/src/tiles/tile-l/styles/index.js]: was just updating an existing tile so prefer to refactor all tiles to use fontFactory in a different PR if we agree to do that.

**Comment by aronshamash** [comment]
@jimhunty rebased so slice layout now has the knobs you added

File: packages/edition-slices/src/slices/listtwoandsixnopic/index.js (hunk 3 above)
Comment (aronshamash) [reply packages/edition-slices/src/slices/listtwoandsixnopic/index.js]: In `shared/responsive-slice.js` the breakpoint is sent as a param to `renderSmall` and `renderMedium`

Comment (jimhunty) [reply packages/edition-slices/src/slices/listtwoandsixnopic/index.js]: Of course it is!

**Pull request merged**